configs/sagan/README.md

```diff
@@ -0,0 +1,59 @@
+| sagan_32  | CIFAR10  |     w/o      |   5    |                       64x1x1                        |    500000    |        [ 9.3217 (400000) ](https://download.openmmlab.com/mmgen/sagan/sagan_cifar10_32_lr2e-4_ndisc5_b64x1_woReUinplace_is-iter400000_20210730_125743-4008a9ca.pth)         |        [ 9.4252 (480000) ](https://download.openmmlab.com/mmgen/sagan/sagan_cifar10_32_lr2e-4_ndisc5_b64x1_woReUinplace_fid-iter480000_20210730_125449-d50568a4.pth )        |        [config](https://github.com/open-mmlab/mmgeneration/tree/master/configs/sagan/sagan_32_woReLUinplace_cifar10_b64x1_lr-2e-4_ndisc5.py)        |       [Log](https://download.openmmlab.com/mmgen/sagan/sagan_cifar10_32_lr2e-4_ndisc5_b64x1_woReUinplace_20210730_125449_fid-d50568a4_is-04008a9ca.json)        |
+| sagan_32  | CIFAR10  |      w       |   5    |                       64x1x1                        |    500000    |        [ 9.2286 (380000) ](https://download.openmmlab.com/mmgen/sagan/sagan_cifar10_32_lr2e-4_ndisc5_b64x1_wReLUinplace_is-iter380000_20210730_124937-c77b4d25.pth)         |        [ 10.7781 (460000) ](https://download.openmmlab.com/mmgen/sagan/sagan_cifar10_32_lr2e-4_ndisc5_b64x1_wReLUinplace_fid-iter460000_20210730_125155-cbefb354.pth)        |        [config](https://github.com/open-mmlab/mmgeneration/tree/master/configs/sagan/sagan_32_wReLUinplace_cifar10_b64x1_lr-2e-4_ndisc5.py)         |        [Log](https://download.openmmlab.com/mmgen/sagan/sagan_cifar10_32_lr2e-4_ndisc5_b64x1_wReLUinplace_20210730_125155_fid-cbefb354_is-c77b4d25.json)        |
+| sagan_128 | ImageNet |     w/o      |   1    |                       64x4x1                        |   1000000    | [ 31.5938 (980000) ](https://download.openmmlab.com/mmgen/sagan/sagan_imagenet1k_128_Glr1e-4_Dlr4e-4_ndisc1_b32x4_woReLUinplace_is-iter980000_20210730_163140-cfbebfc6.pth) | [ 34.7838 (950000) ](https://download.openmmlab.com/mmgen/sagan/sagan_imagenet1k_128_Glr1e-4_Dlr4e-4_ndisc1_b32x4_woReLUinplace_fid-iter950000_20210730_163431-d7916963.pth) | [config](https://github.com/open-mmlab/mmgeneration/tree/master/configs/sagan/sagan_128_woReLUinplace_imagenet1k_b64x4_Glr-1e-4_Dlr-4e-4_ndisc1.py) | [Log](https://download.openmmlab.com/mmgen/sagan/sagan_imagenet1k_128_Glr1e-4_Dlr4e-4_ndisc1_b32x4_woReLUinplace_20210730_163431_fid-d7916963_is-cfbebfc6.json) |
+
+'\*' Iteration counting rule in our implementation is different from others. If you want to align with other codebases, you can use the following conversion formula:
+```
+Total_iters* = Total_iter / (n_disc * n_grad_accu)
```

Comment:
You should tell users what the meaning of `total_iters*` and `total_iters` is.

Total --> total